.github/workflows/industrial_ci_action.yml

```diff
@@ -12,10 +17,19 @@ jobs:
         env:
           - {ROS_DISTRO: foxy, ROS_REPO: main}
           - {ROS_DISTRO: foxy, ROS_REPO: testing}
+          - {ROS_DISTRO: rolling, ROS_REPO: main}
+          - {ROS_DISTRO: rolling, ROS_REPO: testing}
+          - {ROS_DISTRO: galactic, ROS_REPO: main}
+          - {ROS_DISTRO: galactic, ROS_REPO: testing}
+
     env:
-      CCACHE_DIR: "${{ github.workspace }}/.ccache"
+      CCACHE_DIR: ${{ github.workspace }}/.ccache
       BASEDIR: ${{ github.workspace }}/.work
-      CACHE_PREFIX: "${{ matrix.env.ROS_DISTRO }}-${{ matrix.env.ROS_REPO }}"
+      CACHE_PREFIX: ${{ matrix.env.ROS_DISTRO }}-${{ matrix.env.ROS_REPO }}
+      UPSTREAM_WORKSPACE: moveit_msgs.repos
```

Comment:
shouldn't this only be used for the distros that don't provide `object_recognition_msgs` yet?

Reply:
That is correct, I will disable this for rolling and foxy. Galactic still needs this until the first Galactic sync.

Reply:
@vatanaksoytezer `object_recognition_msgs` is now available for galactic and rolling

Reply:
Rolling is synced, and it is also available on galactic testing. But galactic main still needs this as galactic hasn't been synced since the first release